typescript-forms/form-values.ts

```diff
@@ -0,0 +1,27 @@
+interface FormElements extends HTMLFormControlsCollection {
+  name: HTMLInputElement;
+  email: HTMLInputElement;
+  message: HTMLTextAreaElement;
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.querySelector('form') as HTMLFormElement;
+
+  if (!form) {
```

Comment:
You should also not have the `DOMContentLoaded` event listener on this element either.